src/web/command_control/client/components/Details.tsx

```diff
@@ -114,6 +128,16 @@ const commands: {[key: string]: CommandInfo} = {
         humanReadableAvailable: "IN_MISSION__*, PRE_DEPLOYMENT__WAIT_FOR_MISSION_PLAN",
         humanReadableNotAvailable: ""
     },
+    resume: {
+        commandType: CommandType.RESUME,
+        description: 'Resume mission',
+        confirmationButtonText: 'Resume Mission',
+        statesAvailable: [
```

Comment:
Should we add statesNotAvailable for consistency with the pause button, or do we even need that property for these btns?